components/select_team/select_team.jsx

```diff
@@ -57,11 +60,24 @@ export default class SelectTeam extends React.Component {
         this.state = {
             loadingTeamId: '',
             error: null,
+            endofTeamsData: false,
         };
     }
 
     componentDidMount() {
-        this.props.actions.getTeams(0, TEAMS_PER_PAGE);
+        this.props.actions.getTeams(0, TEAMS_PER_PAGE, true);
+    }
+
+    fetchMoreTeams = () => {
+        const {actions, totalTeamsCount} = this.props;
+        const BATCH_OF_TEAMS_TO_LOAD = 4; // Load 4 extra teams
+        const NEW_TEAMS_PER_PAGE = this.props.listableTeams.length + BATCH_OF_TEAMS_TO_LOAD;
+        actions.getTeams(0, NEW_TEAMS_PER_PAGE, true);
```

Comment:
This looks we are always calling page 0 and requesting few more teams? Could we keep track of page count here instead?

Reply:
@sudheerDev I am currently changing this implementation...

Reply:
@sudheerDev I have changed the implementation to use page count to load more teams